sources/academy/platform/get_most_of_actors/store_basics/how_actor_monetization_works.md

```diff
@@ -46,6 +46,29 @@ Monetizing your Actor on the Apify platform involves several key steps:
   - Dataset storage: $1 per 1,000 GB-hours
 - _Example_: you set a $1 per 1,000 results price. Two users generate 50,000 and 20,000 results, paying $50 and $20, respectively. If platform costs are $5 and $2, your profit is $49.
 
+### Pay-per-event pricing model
+
+- _How it works_: you charge users based on specific events triggered programmatically by your Actor's code. You earn 80% of the revenue minus platform costs.
+- _Event cost example_: you set the following events for your Actor:
+  - `Actor start per 1 GB of memory` at 0.005$
+  - `Pages scraped` at 0.002$
+  - `Page opened with residential proxy` at 0.002$ - this is on top of `Pages scraped`
+  - `Page opened with a browser` at 0.002$ - this is on top of `Pages scraped`
+- _Example_:
+  - User A:
+    - Started the Actor 10 times = 0.05$
+    - Scraped 1000 pages = 2.00$
+    - 500 of those were scraped using residential proxy = 1.00$
+    - 300 of those were scraped using browser = 0.90$
```

Comment:
Important: the math is wrong 300 * 0.002 = $0.6.

Also, it would be good to show these calculation explicitly imho.